flask/main.py

```diff
@@ -150,6 +150,13 @@ def organization():
 def connect():
     return "flask /connect"
 
+@app.route('/product/0/info', methods=['GET'])
+def product_info():
+    import time
+    time.sleep(.55)
+    response = make_response(generateMetrics(), 200)
+    response.mimetype = "text/plain"
```

Comment:
If `make_response(...)` on the previous line is what sends the response, then is `response.memtype` doing anything after the fact? Should it be before `response = make_response(...)`? Or if you could message me the code example you found this on, I can take a look.

Reply:
So I'm pretty sure I remember seeing a 200 response from my initial tests. However, today I'm getting back 500s regardless of what Flask is set to (tried with both 200 and 500). Regardless of what the response should be, the N+1 API call still works so, from my reading of the docs and the behavior I'm seeing, the response doesn’t matter only that the call fulfills the detection criteria.